x-pack/test/rbac_api_integration/config.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import path from 'path';
+import { resolveKibanaPath } from '@kbn/plugin-helpers';
+import { EsProvider } from './services/es';
+
+export default async function ({ readConfigFile }) {
+
+  const config = {
```

Comment:
Are there any settings that this suite expects to be set in a particular way that we should assert on? I'm concerned about unexpected changes to configuration files that don't appear to be related to this suite at first glance.

Reply:
Completely agree, and these functionality test configurations are quickly becoming super complicated and interdependent. They're following suit of our other x-pack integration configs, for example https://github.com/elastic/kibana/blob/master/x-pack/test/saml_api_integration/config.js, as it's our only method of code-sharing at the moment. 

The Kibana OSS functional tests are starting to split some stuff out into common here: https://github.com/elastic/kibana/tree/master/test/common but we haven't attacked this yet for x-pack.

I don't know of a good way to explicitly assert that the configs that we're relying upon don't change, but changes to these files are likely to ripple out and cause test failures in other "functional test suites"